web/src/gear_menu.js

```diff
@@ -138,18 +139,20 @@ export function initialize() {
     $("#navbar-buttons").html(rendered_gear_menu);
 }
 
-export function open() {
-    $("#settings-dropdown").trigger("click");
-    // there are invisible li tabs, which should not be clicked.
-    $("#gear-menu").find("li:not(.invisible) a").eq(0).trigger("focus");
-}
-
-export function is_open() {
-    return $(".dropdown").hasClass("open");
-}
-
-export function close() {
-    if (is_open()) {
-        $(".dropdown").removeClass("open");
-    }
-}
+$("#settings-dropdown").on("click", () => {
+    const template = document.querySelector('#gear-menu-toggle')
+      const container = document.createElement('div');
+      container.append(document.importNode(template.content, true));
+
+    tippy('.settings-dropdown', {
+      content: container.innerHTML,
+      trigger: 'click',
+      allowHTML: true,
+      placement: 'bottom-end',
+      arrow: true,
+      animation: 'scale',
+      duration: [250, 200],
+      distance: 12,
+      interactive: true,
+    });
+});
```

Comment:
@amanagr I am trying to migrate gear_menu from ```bootstrap.js``` to ```popover_menu.js```. And think this code should be added to popover_menu.js, but not sure where to do so. Simply adding the function doesn't work, so I am not sure where to add this code in the file. Could you please guide me.